src/style.css

```diff
@@ -1,3 +1,153 @@
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
```

Comment:
i think this file is redundant, you can remove this file

Reply:
+